### ISSUE TITLE
html color tag support

### ISSUE THREAD

**Issue by yalov** [ihsoft/KSPDev_LogConsole#8]
it's pretty popular:
``` c#
private static string PREFIX = "<color=#some>ModName:</color> ";

public static void Log<T>(T msg)
{
    Debug.Log(PREFIX + DateTime.Now.ToString("hh:mm:ss.f ") + msg.ToString());
}
```

**Comment by ihsoft** [comment]
It's actually an interesting idea. Not sure if I'll implement it exactly like this, but adding colors to the logs could be a good idea. It's worth considering.

**Comment by yalov** [comment]
it's already there :)

![](https://i.imgur.com/h71qk4J.png)

**Comment by ihsoft** [comment]
Right, but you gave me an idea to (somehow) use colors in `LogConsole` :) E.g. for distinguishing system/stock vs third-party logs.

As for the standard GUI tags, they all work in the console (I'm using the stock GUI methods to present the data). Not in the files, though.
